src/test/kotlin/io/bazel/kotlin/integration/WriteWorkspace.kt

```diff
@@ -0,0 +1,364 @@
+package io.bazel.kotlin.integration
+
```

Comment:
There is a lot of stuff here - at what point does it make sense to pull in something like [bazel-integration-testing](https://github.com/bazelbuild/bazel-integration-testing/blob/master/java/README.md) and then just have some lightweight kotlin extensions on those methods so you can write in the style of MultiplexTest...

Reply:
Soon, I think. This was a first pass.